package/test/editDocs.js

```diff
@@ -0,0 +1,35 @@
+import fs from "fs";
```

Comment:
Why is this file in the test folder ?

Reply:
Its to change the docs by
- Editing the header to say "Welcome to the docs" and "join the discord server for help" etc
- Removing unnecessary types. typedoc like to do this:
```
ascent        | { large: string, small: string }
ascent.large  | string
ascent.small  | string
```
Instead of just
```
ascent        | { large: string, small: string }
```
So thats what the second part of the code is.

I figured this would be better than having to manually update / edit the docs every time they were compiled, so the doc script in the package.json also runs this automatically after.